CHANGELOG.md

```diff
@@ -273,7 +362,7 @@ Note: The summary of changes represents the diff between v1.8.5 and v1.9.0
 * cmd: Allow to filter metrics with regexp (#12590, @mrostecki)
 * Create healthz HTTP endpoint for kube-proxy replacement (#11733, @soumynathan)
 * datapath: Decouple IPV4_MASQUERADE from IPV4_NODEPORT (Backport PR #13751, Upstream PR #13606, @brb)
- cilium can correctly differentiate protocols between services. (#12628, @nathanjsweet)and port), so that 
+* Differentiate load-balancer keys in the datapath by protocol (in addition to address and port), so that cilium can correctly differentiate protocols between services. (#12628, @nathanjsweet)
```

Comment:
I fixed the old changelog because it has some mangled text. I think the release tool does not handle parenthesis inside the release notes well. I'll take a look at it.

Reply:
I tried to reproduce this by adding a case in the release tool unit tests with this very text and it works. Must have been mangled some other way, maybe human error.